curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-199-iterative-circle-packing.english.md

```diff
@@ -42,12 +45,12 @@ tests:
 <div id='js-seed'>
 
 ```js
-function euler199() {
+function iterativeCirclePacking(n) {
   // Good luck!
   return true;
 }
 
-euler199();
+iterativeCirclePacking(10);
```

Comment:
Can we remove the call to the function?

Reply:
I am in favor of removing it, but since all the other challenges have it still (as far as I can tell), I vote to leave it for now and try to remove them all with a separate issue.